app/helpers/application_helper.rb

```diff
@@ -15,13 +15,13 @@ def faces
   end
 
   def themes
-    Dir['app/assets/stylesheets/themes/*.css.scss']
+    Dir['app/assets/stylesheets/themes/*.scss']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.